doc/parted.md

```diff
@@ -0,0 +1,71 @@
+# parted And Partition Alignment
+
+## General
+
+- parted will try to align newly created partitions. But only if the partition
+start/end are a bit 'fuzzy'. For example:
+
+ - 'parted p ext2 1% 2%' will give nicely aligned partitions; so does
+ - 'parted p ext2 1MB 2MB'. But the supposedly more correct
+ - 'parted p ext2 1MiB 2MiB' will **not** do what you (well, at least I did) expect. The partition size will be 1 MiB + 1 sector.
```

Comment:
Yes, this should be noted in the manpage. {M,G,T}iB addresses are taken as exact. If it is problematic (except missing documentation), we might discuss how to improve it.